src/defines.asm

```diff
@@ -110,6 +110,7 @@
 !sram_save_has_set_rng = $70201E
 !sram_preset_category = $702020
 !sram_room_strat = $702022
+!sram_hexstyle = $702028
```

Comment:
You skipped a couple slots here

Reply:
I have some settings in custom builds that use 24 and 26. I'll move some stuff around to fix that.

I'd like to go through that entire file and reorganize, but I don't want to mix that in with other changes.